restli-int-test-server/src/main/java/com/linkedin/restli/examples/greetings/server/AssociationsAssociationsResource.java

```diff
@@ -0,0 +1,156 @@
+/*
+   Copyright (c) 2021 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.restli.examples.greetings.server;
+
+
+import com.linkedin.restli.common.CompoundKey;
+import com.linkedin.restli.common.HttpStatus;
+import com.linkedin.restli.examples.greetings.api.Empty;
+import com.linkedin.restli.examples.greetings.api.Message;
+import com.linkedin.restli.examples.greetings.api.MessageCriteria;
+import com.linkedin.restli.examples.greetings.api.Tone;
+import com.linkedin.restli.server.BatchFinderResult;
+import com.linkedin.restli.server.BatchPatchRequest;
+import com.linkedin.restli.server.BatchUpdateRequest;
+import com.linkedin.restli.server.BatchUpdateResult;
+import com.linkedin.restli.server.CollectionResult;
+import com.linkedin.restli.server.CreateResponse;
+import com.linkedin.restli.server.PagingContext;
+import com.linkedin.restli.server.RestLiServiceException;
+import com.linkedin.restli.server.UpdateResponse;
+import com.linkedin.restli.server.annotations.AssocKeyParam;
+import com.linkedin.restli.server.annotations.BatchFinder;
+import com.linkedin.restli.server.annotations.Finder;
+import com.linkedin.restli.server.annotations.Key;
+import com.linkedin.restli.server.annotations.Optional;
+import com.linkedin.restli.server.annotations.PagingContextParam;
+import com.linkedin.restli.server.annotations.QueryParam;
+import com.linkedin.restli.server.annotations.RestLiAssociation;
+import com.linkedin.restli.server.resources.AssociationResourceTemplate;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static com.linkedin.restli.examples.AssociationResourceHelpers.DB;
+import static com.linkedin.restli.examples.AssociationResourceHelpers.SIMPLE_COMPOUND_KEY;
+
+
+/**
+ * Association resource under a parent association resource
+ */
+@RestLiAssociation(name="associationsassociations",
+    parent = AssociationsResource.class,
+    namespace = "com.linkedin.restli.examples.greetings.client",
+    assocKeys={@Key(name="anotherSrc", type=String.class),
+        @Key(name="anotherDest", type=String.class)})
+public class AssociationsAssociationsResource extends AssociationResourceTemplate<Message>
```

Comment:
Hmm, I think this is an example, so it is nit, I would suggest to rename from `AssociationsAssociationsResource` to something else. Maybe `AnotherAssociationResource`? Anyway, it may not be better, but the idea would be to have better example naming, I think the Greetings related name would make a good example name. It is your call.
And quite frankly, the parent resource naming is not very ideal too.

Reply:
Won't change since the naming suppose to tell the parental relationship.